person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Good job so far :clap: But from project requirements, a method for adding rentals should be present. I would like to recommend you follow this [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). The book should be sent as a parameter to create the new rental achieving the 'has-many' association :smile:  Please make sure to implement your project. It is very important to follow project requirement to make your app works as expected :+1:

Reply:
The method `add_rental` has been added